transform/transformers/PCKTransformer.py

```diff
@@ -117,11 +114,20 @@ def populate_period_data(self):
                 end_date = datetime.strptime(self.response['metadata']['ref_period_end_date'], "%Y-%m-%d")
                 self.data['12'] = end_date.strftime("%d/%m/%Y")
 
+    def round_currency_values(self):
+        """For RSI Surveys, round the values of the currency fields.
+        Rounds up if the value is .5
+        """
+        if self.survey['survey_id'] == '023':
+            questions = ["20", "21", "22", "23", "24", "25", "26", "27"]
+            self.data.update(
+                {k: str(int(Decimal(self.data[k]).quantize(Decimal('1.'), ROUND_HALF_UP))) for k in questions if k in self.data}
```

Comment:
Cleaner to not all be inline? The logic of what's being looped over is away at the end of the line whilst the eye scans down the left columns.

Does this not trip line length warnings?